pyproject.toml

```diff
@@ -72,38 +72,61 @@ build-backend = "poetry.core.masonry.api"
 
 
 [tool.isort]
```

Comment:
Many of the options specified here were cargo-culted and did not do anything, or were done poorly. This is cleaned up to be as simple as I can manage given my experience with the tooling.